zmsdldb/bin/dldb-helpers.php

```
@@ -154,16 +148,26 @@ public function checkAndCreateBackup($newFiles)
+    }
 
-        return $backupRequired;
+    public function copyDownloadedTempFilesToDestinationAndUnlink(array $filenameToTempPath): void
+    {
+        foreach ($filenameToTempPath as $filename => $tempFile) {
+            $destFile = $this->destinationPath . '/' . $filename;
+            if (!@copy($tempFile, $destFile)) {
+                echo $this->cli->red("Error: Failed to write {$filename} to destination\n\n");
+                continue;
+            }
+            `@unlink`($tempFile);
```

PHPStan (2.1.46)

[error] Syntax error, unexpected '(' on line 161


(phpstan.parse)